pyvenv.cfg

```diff
@@ -1,3 +1,3 @@
-home = C:\Users\papke\AppData\Local\Programs\Python\Python39
```

Comment:
I'd say just add this file to the gitignore. It's always going to be specific to the end user so we shouldn't have it in the repo.

Reply:
Yeah I added it to gitignore but didn't bother removing it, I will do that